packages/core/README.md

```diff
@@ -0,0 +1,9 @@
+# @frctl/core
+
+The core module of [Fractal](http://github.com/frctl/fractal).
```

Comment:
It would be great to shortly describe what core and web modules are dedicated to, that would help some to understand the architecture better.

Reply:
Agreed, but there's no clear separation / idea behind it right now except that they're modular enough so that lerna would not give us circular dependency warnings :) And honestly I could not describe the intent of these modules right now even if I wanted to - there needs to be a whole lot work done before I can get my head wrapped around how Fractal has been built and _why_ exactly this way.